src/interface/orthnull.jl

```diff
@@ -19,43 +19,47 @@ end
 """
-    left_orth(A; [kind::Symbol, atol::Real=0, rtol::Real=0, alg]) -> V, C
-    left_orth!(A, [VC]; [kind::Symbol, atol::Real=0, rtol::Real=0, alg]) -> V, C
+    left_orth(A; [kind::Symbol, trunc, alg_qr, alg_polar, alg_svd]) -> V, C
+    left_orth!(A, [VC]; [kind::Symbol, trunc, alg_qr, alg_polar, alg_svd]) -> V, C
 
 Compute an orthonormal basis `V` for the image of the matrix `A` of size `(m, n)`,
 as well as a  matrix `C` (the corestriction) such that `A` factors as `A = V * C`.
 The keyword argument `kind` can be used to specify the specific orthogonal decomposition
-that should be used to factor `A`, whereas `atol` and `rtol` can be used to control the
+that should be used to factor `A`, whereas `trunc` can be used to control the
 precision in determining the rank of `A` via its singular values.
 
+`trunc` can either be a truncation strategy object or a NamedTuple with fields
+`atol`, `rtol`, and `maxrank`.
+
 This is a high-level wrapper and will use one of the decompositions
-`qr!`, `svd!`, and `left_polar!` to compute the orthogonal basis `V`, as controlled
+`qr_compact!`, `svd_compact!`/`svd_trunc!`, and `left_polar!` to compute the orthogonal basis `V`, as controlled
 by the keyword arguments.
 
 When `kind` is provided, its possible values are
 
-*   `kind == :qrpos`: `V` and `C` are computed using the positive QR decomposition.
-    This requires `iszero(atol) && iszero(rtol)` and `left_orth!(A, [VC])` is equivalent to
+*   `kind == :qr`: `V` and `C` are computed using the QR decomposition.
+    This requires `isnothing(trunc)` and `left_orth!(A, [VC])` is equivalent to
     `qr_compact!(A, [VC], alg)` with a default value `alg = select_algorithm(qr_compact!, A; positive=true)`
 
-*   `kind == :qr`: `V` and `C` are computed using the QR decomposition,
-    This requires `iszero(atol) && iszero(rtol)` and `left_orth!(A, [VC])` is equivalent to
-    `qr_compact!(A, [VC], alg)` with a default value `alg = select_algorithm(qr_compact!, A)`
-
 *   `kind == :polar`: `V` and `C` are computed using the polar decomposition,
-    This requires `iszero(atol) && iszero(rtol)` and `left_orth!(A, [VC])` is equivalent to
+    This requires `isnothing(trunc)` and `left_orth!(A, [VC])` is equivalent to
     `left_polar!(A, [VC], alg)` with a default value `alg = select_algorithm(left_polar!, A)`
 
-*   `kind == :svd`: `V` and `C` are computed using the singular value decomposition `svd_trunc!`,
-    where `V` will contain the left singular vectors corresponding to the singular values that
-    are larger than `max(atol, rtol * σ₁)`, where `σ₁` is the largest singular value of `A`.
-    `C` is computed as the product of the singular values and the right singular vectors,
-    i.e. with `U, S, Vᴴ = svd_trunc!(A)`, we have `V = U` and `C = S * Vᴴ`.
+*   `kind == :svd`: `V` and `C` are computed using the singular value decomposition `svd_compact!`
+    if no truncation is specified through the `trunc` keyword argument or `svd_trunc!`
```

Comment:
How about:
```
`V` and `C` are computed using the singular value decomposition `svd_trunc!` when a truncation strategy is specified using the `trunc` keyword argument, and using `svd_compact!` otherwise.